common/sgx/endorsements.h

```diff
@@ -10,6 +10,17 @@
 
 OE_EXTERNC_BEGIN
 
+#ifndef sgx_qvl_supplemental_t
+typedef struct _sgx_qvl_supplemental_t
+{
+    uint32_t version;
+    time_t earliest_issue_date;
+    time_t latest_issue_date;
+    time_t earliest_expiration_date;
+    time_t tcb_level_date_tag;
+} sgx_qvl_supplemental_t;
```

Comment:
Now that we kept using `oe_get_sgx_quote_validity()` to get validity period instead of using `sgx_qvl_supplemental_t->earliest_expiration_date`, do we still need this struct for this PR?

Reply:
Fixed.